scripts/generate_pdf.py

```
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+"""
+Standalone script to generate PDF from a markdown file using pandoc
+"""
+
+import os
+import subprocess
+import sys
+import re
```

Ruff (0.8.2)

`re` imported but unused

Remove unused import: `re`

(F401)